src/config/Hit.js

```diff
@@ -5,6 +5,7 @@ import './Hit.scss';
 
 export function Hit({ hit, insights, view }) {
   return (
+    /* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
Could we try moving the `onClick` handler on the anchor and see if it works?

Reply:
This works, although it revealed an issue with `search-insights`, so I reverted it to the last working version in another commit.